lib/reversetunnel/srv.go

```diff
@@ -1126,26 +1126,26 @@ func newRemoteSite(srv *server, domainName string, sconn ssh.Conn) (*remoteSite,
 }
 
 // isPreV8Cluster checks if the cluster is older than 8.0.0.
-func isPreV8Cluster(ctx context.Context, conn ssh.Conn) (bool, error) {
+func isPreV8Cluster(ctx context.Context, conn ssh.Conn) (bool, string, error) {
```

Comment:
Can we add an explanation what the `bool` and `string` are? Without reading the function body it's not obvious that the `string` is the version.